third_party/akri/update-akri.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+# fetch the latest version
+helm repo add akri-helm-charts https://project-akri.github.io/akri/ 
+helm repo update akri-helm-charts
+helm pull akri-helm-charts/akri --untar
+
+# Update crds
+cp -r ./akri/crds/* .
+
+# Edit helm-charts
+sed -i 's/v2/v1/' ./akri/Chart.yaml
```

Comment:
the helm_template rule has helm_version arg, where you can select '3'. If that is watch you are patching here.

Reply:
In `akri/Chart.yaml` the apiVersion is mentioned as `v2`. However, our build rule enforces that the apiVersion must be `v1`. This line patches the apiVersion to be `v1`.

Reply:
Updated.